app/Http/Controllers/Order/InvoiceController.php

```diff
@@ -208,10 +208,21 @@ public function getInvoices(Request $request)
                             $sql = 'number like ?';
                             $query->whereRaw($sql, ["%{$keyword}%"]);
                         })
-                        ->filterColumn('status', function ($query, $keyword) {
-                            $sql = 'status like ?';
-                            $query->whereRaw($sql, ["%{$keyword}%"]);
-                        })
+                       ->filterColumn('status', function ($query, $keyword) {
+                           if ($keyword == 'Paid') {
+                               $sql = 'status like ?';
+                               $sql2 = 'success';
+                               $query->whereRaw($sql, ["%{$sql2}%"]);
+                           } elseif ($keyword == 'Unpaid') {
+                               $sql = 'status like ?';
+                               $sql2 = 'pending';
+                               $query->whereRaw($sql, ["%{$sql2}%"]);
+                           } elseif ($keyword == 'Partiallypaid' || $keyword == 'Partially') {
+                               $sql = 'status like ?';
+                               $sql2 = 'partially paid';
+                               $query->whereRaw($sql, ["%{$sql2}%"]);
+                           }
+                       })
```

Comment:
need to refactor whole block of code, 

Can't we use some thing like  below : 
                     ->filterColumn('status', function ($query, $keyword) {
                           $query->where('status','LIKE', '%'.$this->fetchKeyWordForQuery($keyword).'%' );
                       })
and 
    private function fetchKeyWordForQuery($keyword = 'Paid'): string
    {
        return match ($keyword){
            'Paid' => 'success',
            'Unpaid' => 'pending',
            'Partiallypaid' || 'Partially' => 'partially paid'
        };
    }